bool line_detected
time time_det
string orientation
float32 error_rad
float32 error_deg